gox/plan.sh

```diff
@@ -0,0 +1,24 @@
+pkg_origin=core
+pkg_name=gox
+pkg_version=master
```

Comment:
gox has no internal versioning scheme, so we will be solely relying on timestamps in this case?
I realize this is likely to be used to build `vault` - but `vault` has binary releases on its own, right?